pkg/controllermanager/controller/controllerregistration/seed_control_test.go

```diff
@@ -107,32 +97,22 @@ var _ = Describe("controllerRegistrationReconciler", func() {
 		}
 		backupBucketList = &gardencorev1beta1.BackupBucketList{
 			Items: []gardencorev1beta1.BackupBucket{
-				*backupBucket1,
 				*backupBucket2,
 				*backupBucket3,
```

Comment:
Nit: as `backupBucket1` is now removed we could rename the other vars to keep the order. 
```suggestion
				*backupBucket1,
				*backupBucket2,
```